command-expansion-server/package-lock.json

```diff
@@ -19,6 +18,7 @@
         "typescript": "^4.5.5"
       },
       "devDependencies": {
+        "@microsoft/tsdoc": "^0.13.2",
```

Comment:
@goetzrrGit / @dyst5422 is `@microsoft/tsdoc` analogous to `javadoc`? If so, are we making use of it / plan on using it?